commit 688f1e4b6d8f792c44bec3a448664fd9e8949964
Author: Will Deacon <will@kernel.org>
Date:   Tue Sep 15 23:00:31 2020 +0100

    arm64: Rename ARM64_HARDEN_BRANCH_PREDICTOR to ARM64_SPECTRE_V2
    
    For better or worse, the world knows about "Spectre" and not about
    "Branch predictor hardening". Rename ARM64_HARDEN_BRANCH_PREDICTOR to
    ARM64_SPECTRE_V2 as part of moving all of the Spectre mitigations into
    their own little corner.
    
    Signed-off-by: Will Deacon <will@kernel.org>

diff --git a/arch/arm64/include/asm/cpucaps.h b/arch/arm64/include/asm/cpucaps.h
index 07b643a70710..348bfcf6c818 100644
--- a/arch/arm64/include/asm/cpucaps.h
+++ b/arch/arm64/include/asm/cpucaps.h
@@ -31,7 +31,7 @@
 #define ARM64_HAS_DCPOP				21
 #define ARM64_SVE				22
 #define ARM64_UNMAP_KERNEL_AT_EL0		23
-#define ARM64_HARDEN_BRANCH_PREDICTOR		24
+#define ARM64_SPECTRE_V2			24
 #define ARM64_HAS_RAS_EXTN			25
 #define ARM64_WORKAROUND_843419			26
 #define ARM64_HAS_CACHE_IDC			27